flang/test/Driver/bbc-mlir-pass-pipeline.f90

```diff
@@ -31,6 +31,7 @@
 
 ! CHECK-NEXT: 'func.func' Pipeline
 ! CHECK-NEXT:   MemoryAllocationOpt
+! CHECK-NEXT:   ConstExtruderOpt
```

Comment:
The indentation here makes this look like a func.func pass, which it is not. I think FileCheck does not check whitespace